src/program/lwaftr/tests/config/test-config-listen.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+## This checks it can listen, send a command and get a
+## response. It only tests the socket method of communicating
+## with the listen command due to the difficulties of testing
+## interactive scripts.
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+
+# Verify we have the "nc" tool, used to communicate with sockets
+# if we don't have it we just have to skip this test.
+which nc &> /dev/null
+if [[ $? -ne 0 ]]; then
+   echo "No 'nc' tool present, unable to run test." 1&>2
+   exit 1
```

Comment:
skipped tests must exit with SKIPPED_CODE value, which is 43 (check other tests for reference).